content/resources/yum_repository/_index.md

```diff
@@ -21,135 +47,126 @@ resource_description_list:
+    gpgcheck                   true, false # default value: true
+    gpgkey                     String, Array
+    http_caching               String
+    include_config             String
+    includepkgs                String
+    keepalive                  true, false
+    make_cache                 true, false # default value: true
+    max_retries                String, Integer
+    metadata_expire            String
+    metalink                   String
+    mirror_expire              String
+    mirrorlist                 String
+    mirrorlist_expire          String
+    mode                       String, Integer # default value: "0644"
+    options                    Hash
+    password                   String
+    priority                   String
+    proxy                      String
+    proxy_password             String
+    proxy_username             String
+    repo_gpgcheck              true, false
+    report_instanceid          true, false
+    repositoryid               String # default value: 'name' unless specified
+    skip_if_unavailable        true, false
+    source                     String
+    sslcacert                  String
+    sslclientcert              String
+    sslclientkey               String
+    sslverify                  true, false
+    throttle                   String, Integer
+    timeout                    String
+    username                   String
+    action                     Symbol # defaults to :create if not specified
+  end
 syntax_properties_list:
-- '`yum_repository` is the resource.'
-- '`name` is the name given to the resource block.'
-- '`action` identifies which steps Chef Infra Client will take to bring the node into
-  the desired state.'
-- '`baseurl`, `clean_headers`, `clean_metadata`, `cost`, `description`, `enabled`,
-  `enablegroups`, `exclude`, `failovermethod`, `fastestmirror_enabled`, `gpgcheck`,
-  `gpgkey`, `http_caching`, `include_config`, `includepkgs`, `keepalive`, `make_cache`,
-  `max_retries`, `metadata_expire`, `metalink`, `mirror_expire`, `mirrorlist`, `mirrorlist_expire`,
-  `mode`, `options`, `password`, `priority`, `proxy`, `proxy_password`, `proxy_username`,
-  `repo_gpgcheck`, `report_instanceid`, `repositoryid`, `skip_if_unavailable`, `source`,
-  `sslcacert`, `sslclientcert`, `sslclientkey`, `sslverify`, `throttle`, `timeout`,
-  and `username` are the properties available to this resource.'
-syntax_full_code_block: null
-syntax_full_properties_list: null
-syntax_shortcode: null
-registry_key: false
-nameless_apt_update: false
-nameless_build_essential: false
-resource_package_options: false
+syntax_full_properties_list:
+- "`yum_repository` is the resource."
+- "`name` is the name given to the resource block."
+- "`action` identifies which steps Chef Infra Client will take to bring the node into
+  the desired state."
+- "``baseurl``, , ``clean_metadata``, ``cost``, ``description``, ``enabled``, ``enablegroups``,
```

Comment:
```suggestion
- "``baseurl``, ``clean_metadata``, ``cost``, ``description``, ``enabled``, ``enablegroups``,
```
You're generating an extra comma.

Reply:
Let me see what in the automation caused that one

Reply:
Turns out the automation got it right and I missed it. That's commited now.